tests/core/usage_statistics/test_usage_statistics_data_context.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import annotations
+
+import inspect
+import logging
+from unittest import mock
+
+import pytest
+
+from great_expectations.core.usage_statistics.events import UsageStatsEvents
+from great_expectations.core.usage_statistics.usage_statistics import ENABLED_METHODS
+
+logger = logging.getLogger(__name__)
+
+
+def filter_enabled_methods(
+    enabled_methods: dict[str, UsageStatsEvents]
+) -> dict[str, UsageStatsEvents]:
+    return {
+        k.split(".")[1]: v
+        for k, v in enabled_methods.items()
+        if k.startswith("AbstractDataContext") and not k.endswith("__init__")
+    }
```

Comment:
I wanted to make this filtered dict a fixture but didn't because I need it for parameterization below - a global dict would work as well but I thought a function was a happy middle ground

Reply:
Do you also need to check that `k.split(".")` has length 2? Is the `1` here suppose to be grabbing the last element?